src/Check/Memcache.php

```diff
@@ -43,7 +44,7 @@ public function __construct($host = '127.0.0.1', $port = 11211)
         $port = (int) $port;
         if ($port < 1) {
             throw new InvalidArgumentException(sprintf(
-                'Invalid port number - expecting a positive integer',
+                'Invalid port number %s - expecting a positive integer',
```

Comment:
Ah yes ofc, altered the error and added UT for the constructor validation specifically. Had to update psalm-baseline though because I'm sending an incorrect $host type.